open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/synchronization/beans/SchemaElement.java

```diff
@@ -0,0 +1,169 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.analyticsmodeling.synchronization.beans;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+/**
+ * <p>
+ *     The SchemaElement object provides a base class for the pieces that make up a schema for an asset.
+ *     A schema provides information about how the data is structured in the asset.  Schemas are typically
+ *     described as nested structures of schema elements.  There are two basic types:
+ * </p>
+ *     <ul>
+ *         <li>SchemaType describes the structure of data.</li>
+ *         <li>SchemaAttribute describes the use of another schema as part of the structure within a bigger schema.</li>
+ *     </ul>
+ * <p>
+ *     Assets are linked to a SchemaType.
+ */
+@JsonAutoDetect(getterVisibility=PUBLIC_ONLY, setterVisibility=PUBLIC_ONLY, fieldVisibility=NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown=true)
+@JsonSubTypes(
+        {
+                @JsonSubTypes.Type(value = SchemaAttribute.class, name = "SchemaAttribute"),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.